libbpf-tools/gethostlatency.h

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+#ifndef __GETHOSTLATENCY_H
+#define __GETHOSTLATENCY_H
+
+#define TASK_COMM_LEN 16
+
+struct val_t {
+	__u32 pid;
+	char comm[TASK_COMM_LEN];
+	char host[80];
+	__u64 ts;
+};
+
+struct data_t {
+	__u32 pid;
+	__u64 delta;
+	char comm[TASK_COMM_LEN];
+	char host[80];
```

Comment:
Not a big deal, but I think you can probably reuse `val_t` and save an extra 2 probe_read(). You can just fixup `ts` to actually hold `delta` before transmitting to userspace.

Reply:
Nice idea. Will do.